### PR TITLE
Story3 add assessment day date event

### PR DESCRIPTION
**Pull request by simonaambr**
Added option for custom assessment day to the applicant form, and added customAssessmentDay field in database. Also made sure the chosen custom assessment day appears in the pop-up of each applicant.

### CODE REVIEW

File: db/academyPortal.sql [hunk 1]
```diff
@@ -573,6 +574,55 @@ VALUES
...
+	(198,35,'444','','2021-09-28 19:31:43',0,X'4E2F41',0,'N/A'),
+	(199,44,'Something something something dark sidegfdfdgfdgdg','','2021-09-28 21:40:15',0,X'4E2F41',1,'2021-09-28 21:40:21'),
+	(200,44,'fdgfdhfdhhdh','','2021-09-28 21:40:17',0,X'4E2F41',1,'2021-09-28 21:40:23'),
+	(201,44,'Something something something dark side','','2021-09-28 21:40:19',0,X'4E2F41',1,'2021-09-28 21:40:22'),
+	(203,51,'walk the dawgs','','2021-09-29 11:57:28',0,X'4E2F41',0,'N/A');
+
+/*!40000 ALTER TABLE `tasks` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
```
Comment (norayna) [inline db/academyPortal.sql:625]: uhmmm what is this new table?

**Review by norayna — CHANGES_REQUESTED** [review]
since I've already made a comment and this PR hasn't gone through flawlessly, i will request another little change:

on the custom assessment day field, both in the form and in the applicant pop-up, i want it to say "Custom assessment day (optional)".

ALSO, i am not clear as to whether the product owner wants this scenario to be allowed:
- NO assessment day chosen
- YES custom assessment day chosen

make sure to ask the PO whether that is something the system should allow. if not, there should be some validation that requires the assessment day from the dropdown to be selected for the custom assessment day to be set

File: db/academyPortal.sql (hunk 1 above)
Comment (MrLukey) [reply db/academyPortal.sql]: Removed

**Review by norayna — CHANGES_REQUESTED** [review]
changes requested

**Comment by norayna** [comment]
oops i didn't mean to request changes on this one, reviewing now

**Review by norayna — CHANGES_REQUESTED** [review]
all changes are fine however it needs backend validation in ApplicantValidator

**Review by norayna — DISMISSED** [review]

**Pull request closed without merging**